src/validation_workflow/validation_args.py

```diff
@@ -0,0 +1,51 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import argparse
+
+from .rpm import Rpm
+from .tar import Tar
+from .yum import Yum
+
+
+class ValidationArgs:
+    SUPPORTED_DISTRIBUTIONS = ["tar", "yum", "rpm"]
+    SUPPORTED_PLATFORMS = ["linux"]
+    DISTRIBUTION_MAP = {"tar": Tar, "yum": Yum, "rpm": Rpm}
+
```

Comment:
A better practice is to write them into multiple lines instead of one line, for better visual or better maintenance.